sql/system.go

```diff
@@ -150,6 +167,7 @@ func addSystemDatabaseToSchema(target *MetadataSchema) {
 	target.AddDescriptor(keys.SystemDatabaseID, &descriptorTable)
 	target.AddDescriptor(keys.SystemDatabaseID, &usersTable)
 	target.AddDescriptor(keys.SystemDatabaseID, &zonesTable)
+	target.AddDescriptor(keys.SystemDatabaseID, &ReportingTable)
```

Comment:
Make sure you run all of the sql tests; I believe you may have to modify some expected data in a few of the SQL logic tests (specifically the file "sql/testdata/system").